vars/cloudFoundryDeploy.groovy

```diff
@@ -196,11 +196,17 @@ private String deleteOptionIfRequired(Map config) {
     }
 }
 
-private String stopOldAppIfRequired(Map config) {
+private void stopOldAppIfRequired(Map config) {
+    String oldAppName = "${config.cloudFoundry.appName}-old"
+    String cfStopOutputFileName = 'cfStopOutput.txt'
```

Comment:
Having one hard coded log file name causes problems when we have parallel executions. I can imaging parallel executions well for `cfDeploy` in order to populate different accounts/spaces. When the parallel executions overwrite the log file we get in trouble. --> I would like to suggest to have some random UUID in the file name (or alternatively some identifier for the account we deploy into).

Reply:
After an discussion with the colleagues we decided to only parse the output instead of RC + output written in the file. 
With that new approach a UUID would be obsolete.

Reply:
Just for clarification: you mean something like
`cf stop $oldAppName  &> |grep -v FAILED`
and using the return code of that statement?

In this case using the ReadFileRule in the test is also obsolete.

Reply:
Almost correct. 
I mean: 

`String x = sh(returnStdout: true script: "cf stop $oldAppName &> |grep -v FAILED")`
`if (!x.contains("$oldAppName not found")`
`error "...."` 

Commit will follow later.

Reply:
>we decided to only parse the output instead of RC

Not sure if ignoring the return code is a cool idea ...
Return codes are the "canonical" way how command line based tools inform about issues they encouter during the journey.